### DISCUSSION TITLE
osu!lazer performance issues/optimization when?

### DISCUSSION THREAD

**Discussion by ameenoacids** [ppy/osu#13468] [Q&A] [answered]
I have an Intel Core i5-7300HQ CPU @ 2.50GHz, 8 GB RAM, Nvidia GTX 1050 win10
It's a Lenovo laptop from 2016 (Legion Y520)

osu stable runs decently for me, turning all the extra quality settings off (things like hit lighting etc) and i get about 300fps semi consistently mid map and it feels good to play. not many fps drops etc
 
however, osu lazer on the other hand i've been wanting to switch to for a long time, but with the lowest settings possible and uncapped fps i can only get 120fps with the same settings. i expected osu lazer to run even better than osu stable or at least be around the same but it doesn't run well at all. i really would enjoy lazer way more and switch to it asap if the game was more optimized for older systems. 

![image](https://user-images.githubusercontent.com/85778989/121766992-42063700-cb5e-11eb-8c15-7114ee7b191e.png)

**Comment by peppy** [comment, marked as answer]
You should NOT be using uncapped fps on such a system. It will work against you.

Please make sure you set single threaded mode if your system is struggling, and frame limiter to 60, 120 or 240. You want stable FPS, not high FPS.

As for your post title "when", this is a very wrong question to ask. Optimisations are ongoing and there is no "when".
